src/libutil/signature/signer.cc

```diff
@@ -0,0 +1,120 @@
+#include "signature/signer.hh"
+#include "network/user-agent.hh"
+#include "util.hh"
+
+#include <curl/curl.h>
+#include <sodium.h>
+
+namespace nix {
```

Comment:
We usually do not indent things within namespaces, since it results in basically the whole file being indented.